database-commons/src/main/java/io/cdap/plugin/db/CommonSchemaReader.java

```diff
@@ -127,4 +128,23 @@ public Schema getSchema(ResultSetMetaData metadata, int index) throws SQLExcepti
 
     return Schema.of(type);
   }
+
+  @Override
+  public boolean ignoreColumn(ResultSetMetaData metadata, int index) throws SQLException {
+    return false;
+  }
+
```

Comment:
Allows to not include a column in the inferred schema. Thus we can support different data types for Sink and Source plugins.

For example, in MS SQL `TIMESTAMP` is the synonym for the `ROWVERSION` data type, values of which are automatically generated. Therefore `TIMESTAMP` can not be supported by Sink plugin.